### ISSUE TITLE
A11y_AzureNotebooks_Jupyter_MenuBar_Keyboard: Dropdowns(New NoteBook,Revert to Checkpoint, Download as) present inside "File" menuitem are  not accessible through keyboard.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [jupyter/notebook#3981]
**EnvironmentDetails:**
Application Name:AzureNotebooks_Jyupter
Login Url: notebooks.azure.com
BrowserDetails:
Edge:Microsoft Edge 41.16299.611.0
Operating System:Windows10

**ReproSteps:**
1. Login with the appropriate credentials to azure notebook account.    
2. Navigate to any of the file names present in the samples-->>clickin the file name-->Welcome page opens with menubar.
3. Start Navigating through Keyboard inside "File" menuitem.
4. Navigate till Dropdowns(New NoteBook,Revert to Checkpoint, Download as) and Verify dropdowns are not accessible through keyboard.

**Actual:**
1. Dropdowns(New NoteBook,Revert to Checkpoint, Download as) present inside "File" menuitem are not accessible through keyboard.
2. Only by Hovering Mouse on Dropdowns(New NoteBook,Revert to Checkpoint, Download as) then only Dropdowns are expanding but by keyboard
dropdowns are not accessible.

**Note:**
Same issue is observed for all Dropdowns which are present inside all menuitems.
Same issue is observed in "**Chrome+NVDA**" Configuration.
Same issue is observed in **Narrator** and **NVDA**.

**Expected:**
1. Dropdowns(New NoteBook,Revert to Checkpoint, Download as) present inside "File" menuitem should be accessible through keyboard.
2. Not only by Mouse hover dropdowns should be accessible but by keyboard also dropdowns should be accessible.

**UserImpact:**
Keyboard Only Users face difficulty if dropdowns are accessible only by mouse hover but not through keyboard.

**Attachment:**
(https://github.com/jupyter/notebook/files/2400313/A11y_AzureNotebooks_Jupyter_MenuBar_Keyboard.pptx)

**Comment by jtpio** [comment]
Closing as there are now different issues tracking accessibility improvements for Jupyter Notebook 7, for example https://github.com/jupyter/notebook/issues/6800

Thanks!
